config/settings/__init__.py

```diff
@@ -1 +1,20 @@
-from .base import *  # noqa
+# from django.utils.version import get_version
```

Comment:
Code scars; remove the commented lines; Also, why are they doing the same thing done in `config/__init__.py`?